### DISCUSSION TITLE
how to remove pinned

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [komikku-app/komikku#215] [General]
when there is no pinned source in global search. maybe autohide it if no pinned source 
it is pain.
kindly can u move these below like how j2k have or previously tachi hab floating fab?
instead of tab

**Comment by cuong-tran** [comment]
The option to use All instead if Pin is now persistent. It should work for you.

**Comment by HikaruIchijyo** [comment]
i have no pinned source but whenever i try to search 
it always selects pinned 
![Screenshot_2024-10-18-10-17-23-497_app.komikku.beta.jpg](https://github.com/user-attachments/assets/dcc5203c-f69b-4214-bc12-1b1eda08ee1d)

**Reply by cuong-tran** [reply]
I see, better make it persistent

**Reply by HikaruIchijyo** [reply]
> I see, better make it persistent 

yeah or make it so if no pinned source remove pinned button and all only show has result
if pinned present then show pinned all has-result & make all persistent untill u select pinned

**Reply by cuong-tran** [reply]
ok, doing both

**Reply by HikaruIchijyo** [reply]
> ok, doing both

thanks

**Comment by cuong-tran** [comment]
it's done

**Reply by HikaruIchijyo** [reply]
thanks will test in next preview
